extensions/ql-vscode/src/databases/local-databases/database-contents.ts

```diff
@@ -1,4 +1,7 @@
+import { pathExists, remove } from "fs-extra";
+import { join } from "path/posix";
```

Comment:
The `join` in `database-fetcher.ts` seems to come from `path`, not `path/posix`:

```suggestion
import { join } from "path";
```